test/unit/services/awsAccess.test.ts

```diff
@@ -0,0 +1,127 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+import {
+  IdentitystoreClient,
+  ListUsersCommand,
+  CreateUserCommand,
+  CreateGroupMembershipCommand,
+} from "@aws-sdk/client-identitystore";
+import {
+  createUser,
+  addUserToGroup,
+  fetchAwsUserIdByUsername,
+} from '../../../utils/awsFunctions';
+
+describe('AWS Identity Store Functions', () => {
+  let sendStub: sinon.SinonStub;
+
+  beforeEach(() => {
+    // Reset all stubs before each test
```

Comment:
can we remove this comment? The code is self explanatory.

Reply:
Sure